### ISSUE TITLE
adding folder fails

### ISSUE THREAD

**Issue by locked-fg** [OneDrive/onedrive-sdk-python#84]
I try to add a folder "foo" into the (existing) folder "/a/b/" with the following code:
```
    i = onedrivesdk.Item()
    i.name = "foo"
    i.folder = onedrivesdk.Folder()
    cs = client.item(path="/a/b/").children
    cs.add(i)
```
Yet I just get:
```
File "C:\Users\info\AppData\Local\Programs\Python\Python35-32\lib\site-packages\onedrivesdk\request\children_collection.py", line 43, in add
    entity = Item(json.loads(self.send(entity).content))
NameError: name 'Item' is not defined
```

**Comment by lucabezerra** [comment]
I've submitted yesterday a pull request that fixes this. In the meantime, you might want to go into your children_collection.py file and add `from ..model.item import Item`.
The file is located under ../onedrive-sdk-python/src/python2/request/children_collection.py.
However, if you're using it inside a virtualenv, you should make the change there, not in the SDK folder you've cloned (if you did so).

**Comment by D4N14L** [comment]
This is fixed in #86 which is now in master. I will update the version and push to pypi.

**Comment by locked-fg** [comment]
Awesome guys! Thanks a lot.
